src/behat/ConfigurationPage.php

```diff
@@ -117,6 +117,11 @@ public function setProperties($properties)
                     $checkboxValue = $checkbox->getValue();
                     if (($value && !$checkboxValue) || (!$value && $checkboxValue)) {
                         try {
+                            // if the parent is td tag then doesn't throw exception
```

Comment:
I don't see the connection between this comment and the if.
What do you want to test exactly ?

Reply:
as you can see here the build is green https://jenkins.centreon.com/view/All/job/centreon-bam/job/BAM-806-fix-acceptance-19.04.x/ because mink try to click on the *td* element it didn't throw an exception.